android/src/main/java/com/openxchange/deltachatcore/DeltaChatCorePlugin.java

```diff
@@ -185,13 +193,13 @@ private void coreListener(MethodCall call, Result result) {
         }
         EventChannelHandler eventHandler;
         if (eventHandlers.indexOfKey(eventId) < 0) {
-            eventHandler = new EventChannelHandler(applicationDcContext, registrar.messenger(), eventId);
+            eventHandler = new EventChannelHandler(nativeInteractionManager, registrar.messenger(), eventId);
             eventHandlers.put(eventId, eventHandler);
         } else {
             eventHandler = eventHandlers.get(eventId);
         }
         if (add) {
-            int newObserverId = eventHandler.addObserver();
+            int newObserverId = eventHandler.addListener();
```

Comment:
Maybe 'newListenerId' is a better wording than 'newObserverId'.

Reply:
Fixed by https://github.com/open-xchange/flutter-deltachat-core/pull/8/commits/5cc17937d9a06c1632d1e6bea958dad2712a3667